docs/deteriorate.rst

```diff
@@ -18,16 +18,16 @@ dwarves feel, your FPS does not like it!
 Usage
 -----
 
-``deteriorate start --types <types> [--freq <frequency>] [--quiet]``
-    Starts deteriorating the specified item types while you play.
+``deteriorate start --types <types> [--freq <frequency>] [--quiet] [--useable]``
+    Starts deteriorating the specified item types while you play, keeping useable corpse pieces.
```

Comment:
```suggestion
    Starts deteriorating the specified item types while you play.
```

Reply:
The effect of the option should be documented in the Options section, and an example using it should go in the Examples section.

Reply:
Okay.